kubemarine/procedures/install.py

```diff
@@ -139,6 +139,19 @@ def system_prepare_system_sysctl(group: NodeGroup):
         sysctl.reload,
     ])
 
+@_applicable_for_new_nodes_with_roles('all')
+def system_prepare_add_etcd_user(group: NodeGroup):
```

Comment:
I think, these changes make sense if run ETCD container as non root user.

Reply:
Checked installation with the changes from this PR, and etcd container is still running as root. Do I correctly understand that it is useless to change ownership of the directory in this case? @pranavcracker Could you shed more lights on what is the reason for changing of the ownership and under what conditions? Also found similar issue with comment https://github.com/rancher/rancher/issues/29311#issuecomment-785699317

Reply:
We haven't made any changes that will affect the container user, thus that is true. All that has changed is a new user named etcd has been created and the ownership of the data directory, which has been changed to `etcd:etcd`. In my opinion, there is no other result if we alter the ownership of these directories other than making the scan successful.

Reply:
In this case I suggest to leave this as known issue and do not fix.

Reply:
We could go back to the solution after the following enhancement implementation [run control-plane as non-root](https://github.com/kubernetes/kubeadm/issues/2473)